src/features/wallet/send/SendDetailsForm.tsx

```diff
@@ -200,16 +236,28 @@ const SendDetailsForm = ({
         isFirst
         defaultValue={address}
         onChange={setAddress}
+        onEndEditing={onDoneEditingAddress}
         label={t('send.address.label')}
         placeholder={t('send.address.placeholder')}
         extra={
           <AddressExtra
             addressLoading={addressLoading}
-            isValidAddress={Address.isValid(address)}
+            isValidAddress={
+              Address.isValid(address) &&
+              isHotspotAddress !== undefined &&
+              !isHotspotAddress
+            }
             onScanPress={onScanPress}
           />
         }
-        footer={<AddressAliasFooter addressAlias={addressAlias} />}
+        footer={
+          <Box>
+            <AddressAliasFooter addressAlias={addressAlias} />
+            {isHotspotAddress && (
+              <Text color="redMain">{t('send.not_valid_address')}</Text>
```

Comment:
There's a few `<Text>` components in this file that are missing the `variant` attribute. This causes them to use the system font. I think we want either "body2"  or "body2Mono" as the variant.